src/LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Diego Juliao, Yossely Mendoza
+Copyright (c) 2020 Diego Juliao, Yossely Mendoza
```

Comment:
2019-2021

Reply:
![image](https://user-images.githubusercontent.com/7026066/109862384-5f0d9e80-7c2e-11eb-94de-07882b9eff49.png)

Reply:
# 🕵️

Reply:
Thanks for this review